packages/plugin-compiler/src/test/resources/sample/input/Sample.kt

```diff
@@ -144,6 +144,9 @@ class Sample : RealmObject {
     val linkingObjectsByList by backlinks(Sample::objectListField)
     val linkingObjectsBySet by backlinks(Sample::objectSetField)
 
+    @PersistedName(name ="persistedLinkingObjects")
+    val linkingObjectsWithPersistedName by backlinks(Sample::objectListField)
```

Comment:
again, not sure if one would want a test for this, this is different from the test that we had so far but is probably still a scenario worth investigating